openedx/core/apidocs.py

```diff
@@ -10,7 +11,7 @@
     version="v1",
     description="APIs for access to Open edX information",
     #terms_of_service="https://www.google.com/policies/terms/",         # TODO: Do we have these?
-    email="oscm@edx.org",
+    email=settings.API_ACCESS_FROM_EMAIL,
```

Comment:
```suggestion
    email=settings.API_ACCESS_MANAGER_EMAIL,
```
I think the intention of the settings is the the API access request would come _from_ the `API_ACCESS_FROM_EMAIL` and be _to_ the `API_ACCESS_MANAGER_EMAIL`.